data/presets/presets/attraction/kiddie_ride.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "temaki-amusement_park",
+    "fields": [
+        "{attraction}",
+        "fee",
+        "payment_multi_fee",
+        "charge_fee"
+    ],
+    "moreFields": [
+        "{attraction}",
+        "max_age",
+        "min_age"
+    ],
+    "geometry": [
+        "point"
```

Comment:
Should this one also have "area" ?

Reply:
I'd agree with the wiki here. Usually, those kiddy rides are just small horses, cars or helicopters moving to and fro. In my opinion they are too small to be mapped as areas.

Reply:
Oh ok!  Didn't realize that's what it was for.